src/demoAskScriptServer.ts

```diff
@@ -14,8 +14,15 @@ import { parse as parseAskScript, AskScriptCode } from './askscript';
 import chalk = require('chalk');
 import { customAlphabet } from 'nanoid';
 
-import packageInfo from './package.json';
+let packageInfo = {
+  version: 'dummy',
+};
 
+try {
+  packageInfo = require('../package.json');
+} catch (e) {
+  packageInfo = require('./package.json');
+}
```

Comment:
Demo ask script server is currently only run as a development tool - we don't need to support it in a run package